public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -116,15 +116,14 @@ export const MegaMenu = memo(
           <ScrollContainer height="100%" overflowX="hidden" showScrollIndicators>
             <ul className={styles.itemList} aria-label={t('navigation.megamenu.list-label', 'Navigation')}>
               {navItems.map((link, index) => (
-                <Stack key={link.text} direction={index === 0 ? 'row-reverse' : 'row'} alignItems="start">
```

Comment:
this Stack caused a11y errors (no divs allowed inside uls). And seems unnnessary as MegaMenuItem is always a single item (li) so this logic of reverse-row never has any effect.

Reply:
ah yeah, this is a hang over from the old docked implementation where the dock button lived in the first row alongside `Home`. nice job ripping it out 👍